serviceconnector/src/tree/IconPath.ts

```diff
@@ -14,6 +14,6 @@ export function getIconPath(iconName: string): TreeItemIconPath {
 function getResourcesPath(): string {
     return ext.ignoreBundle ?
         path.join(__dirname, '..', '..', '..', 'resources') :
-        path.join(__dirname, 'node_modules', '@microsoft', 'vscode-azext-azureutils', 'resources');
+        path.join(__dirname, 'node_modules', '@microsoft', 'vscode-azext-serviceconnector', 'resources');
```

Comment:
Shouldn't this be an additional line, not a replacement?

Reply:
This getResourcesPath is specifically for service connector resources. The azure package has a seperate IconPath.ts file that handles its resources, so AFAIK it shouldn't need the additional line.